### PR TITLE
Update Paragon to 13 and make the app brandable

### PR DESCRIPTION
**Pull request by davidjoy**
This PR is best reviewed commit-by-commit.  It includes some work to fix the linter and properly set the NODE_ENV that I wanted to fix as well since we rarely touch this MFE.  

The linting commit is a lot of changes, but they're all just formatting things for the most part.  The only one that isn't is the change to `<option value="" />` to give it an explicit `&nbsp;` of content. 

Screen Shot 2021-01-26 at 10 36 31 AM

### CODE REVIEW

File: src/profile/__snapshots__/ProfilePage.test.jsx.snap [hunk 1]
```diff
@@ -1238,7 +1209,7 @@ exports[`<ProfilePage /> Renders correctly in various states while saving an edi
     className="profile-page-bg-banner bg-primary d-none d-md-block p-relative"
   />
   <div
-    className="container-fluid"
+    className="container"
```
Comment (abutterworth) [inline src/profile/__snapshots__/ProfilePage.test.jsx.snap:1212]: Are these changes to container instead of container-fluid desired?

Comment (davidjoy) [reply src/profile/__snapshots__/ProfilePage.test.jsx.snap:1212]: Yes... the app had a max-width before, so this preserves that.  I mean, I could put it back to fluid, but it makes it super wide.  I don't feel strongly; was just trying to keep it 'the same'

File: src/profile/forms/Country.jsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ class Country extends React.Component {
                     value={country}
                     onChange={this.handleChange}
                   >
-                    <option value="" />
+                    <option value="">&nbsp;</option>
```
Comment (abutterworth) [inline src/profile/forms/Country.jsx:87]: Intended?

Comment (davidjoy) [reply src/profile/forms/Country.jsx:87]: Yeah - the linter didn't like it having some textual content.

File: src/profile/forms/elements/EditButton.jsx [hunk 3]
```diff
@@ -13,7 +13,8 @@ function EditButton({
 }) {
   return (
     <Button
-      className={classNames('btn-sm btn-link', className)}
+      variant="link"
+      className={classNames('btn-sm', className)}
```
Comment (abutterworth) [inline src/profile/forms/elements/EditButton.jsx:17]: Button has a `size="sm"` prop: https://react-bootstrap.github.io/components/buttons/#sizes

**Review by abutterworth — APPROVED** [review]
A couple sanity checks, but LGTM

File: src/profile/forms/elements/EditButton.jsx (hunk 3 above)
Comment (davidjoy) [reply src/profile/forms/elements/EditButton.jsx:17]: Oh, cool, I'll use that.

**Pull request merged**